supergrub-mkcommon

```diff
@@ -66,21 +66,52 @@ function generate_and_copy_mo_files () {
 }
 
 function check_unifont () {
+  local unifont_basename="unifont.pf2"
+  local unifont_destdir="$overlay/boot/grub/fonts"
+  local unifont_destfile="$unifont_destdir/$unifont_basename"
+  mkdir -p -- "$unifont_destdir" || return $?
+
+  local font_dirs="$(echo $(echo '
+    /boot/grub/fonts
+    /usr/share/grub/fonts
+    '))"
+  local dir= ext=
+  for dir in $font_dirs; do
+    if [ -e "$dir/$unifont_basename" ]; then
+      cp -- "$dir/$unifont_basename" "$unifont_destdir"/; return $?
+    fi
+    if [ -e "$dir/$unifont_basename".gz ]; then
+      <"$dir/$unifont_basename".gz gzip -d > "$unifont_destfile"; return $?
+    fi
+  done
 
   # Find unifont font file to create grub font. This is needed for gfxterm in grub, which
   # in turn is needed for displaying non-ASCII characters for non-English translations.
   # This unifont finding code was copied from grub's configure.ac.
-  for ext in pcf pcf.gz bdf bdf.gz ttf ttf.gz; do
-    for dir in . /usr/src /usr/share/fonts/X11/misc /usr/share/fonts/unifont; do
-      if test -f "$dir/unifont.$ext"; then
-        font_source="$dir/unifont.$ext"
-        break 2
-      fi
+  local font_exts="$(echo $(echo '
+    bdf
+    pcf
+    ttf
+    '))"
```

Comment:
Idem

Reply:
I just show this request. I propose something related to the font loading (in menus only) in [this](https://github.com/supergrub/supergrub/issues/87).

In Archlinux the unicode font file name is `unicode.pf2` and is in the grub package files at `/usr/share/grub/`, while it is also copied to `$prefix/fonts`.

Reply:
Better approach for diff flickering:
```
  local font_exts="\
    bdf \
    pcf \
    ttf \
"
```

Reply:
Sorry for finding this late. Github somehow doesn't notify me about code discussions. :-(
I'll [repeat my comment here](https://github.com/supergrub/supergrub/commit/c36ecd0ed62faf0e05bc2250f7c2df4674213319#r142632666) in case someone finds this via search engine:
You can omit these backslashes. For feeding into "for", the newlines in additions to the spaces do not make a difference.